### PR TITLE
Refs #10970 - Rails4 master merge

### PR DESCRIPTION
**Pull request by johnpmitsch**
Updating Rails 4 branch so we can open up the PR of Rails4 > master

### CODE REVIEW

**Comment by daviddavis** [comment]
ACK?

File: db/seeds.d/106-mail_notifications.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  {:name              => :katello_promote_errata,
+   :description       => N_('A post-promotion summary of hosts with installable errata'),
+   :mailer            => 'Katello::ErrataMailer',
+   :method            => 'promote_errata',
+   :subscription_type => 'alert'
+  }
+]
+
+notifications.each do |notification|
+  ::MailNotification.where(:name => notification[:name]).first_or_create
```
Comment (johnpmitsch) [inline db/seeds.d/106-mail_notifications.rb]: This is like this in master, I am not sure why we are only creating MailNotifications with just a name, and not all the other attributes?

**Comment by cpeters** [comment]
ACK

**Comment by komidore64** [comment]
looks good to me. i don't see any glaring things that'd break rails4.

**Comment by johnpmitsch** [comment]
[test]

**Pull request merged**